src/isValidDateAfter.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+
+export default function isValidDateAfter(outstandingTask) {
+  const currentDateTime = moment();
+  const targetDeploymentDateTime = moment(outstandingTask.targetDeploymentDateTime, 'YYYY-MM-DD HH:mm Z');
+  console.log(`with targetDeploymentDateTime: ${targetDeploymentDateTime.format()}.`);
+  return currentDateTime.isAfter(targetDeploymentDateTime) && !outstandingTask.__isDeployed;
```

Comment:
Please rename `__isDeployed` to `isDeployed` and omit from this file altogether. The consumer can access isDeployed from outstandingTask directly if required.